projects/dynamicforms/src/components/upload/upload.component.ts

```diff
@@ -21,6 +26,8 @@ import { FormDataService } from '../../core/services';
   templateUrl: './upload.component.html',
 })
 export class UploadComponent extends AbstractFormComponent implements OnInit {
+  protected readonly fileSizeSymbols = ['Bytes', 'KB', 'MB'];
```

Comment:
I would say that this property is really specific to `convertFileSize` method. I don't see how it could be reused somewhere else in the future. I would move it back to `convertFileSize` method. No need to "pollute" global class scope.

Reply:
If a customer wants to change the way of showing the size of documents, for example, to display only in KB, he/she would need to override the entire method (basically duplicate code and change only this list inside of a method).
With this approach, it is enough just to redeclare fileSizeSymbols property.

Reply:
In that case customer would again need to override the `convertFileSize` method since the logic for choosing file size is in there. For instance, if customer chooses to have only KBs, and the actual file is 16 Bytes, he/she would get **16KB** as a result, which is incorrect.